src/uwtools/set_config.py

```diff
@@ -72,6 +72,21 @@ def parse_args(argv):
         action='store_true',
         help='If provided, print a list of required configuration settings to stdout',
     )
+
+    parser.add_argument(
+        '--convert_input_file',
```

Comment:
Totally a nit-pick...could these be named as "input_file_type"? Then later, the decision point could be to use the flag if the flag is provided or to try to infer it from the file name suffix if it isn't provided.